hazelcast/src/main/java/com/hazelcast/map/IMap.java

```diff
@@ -452,6 +452,8 @@
      * @param predicate matching entries with this predicate will be removed
      *                  from this map
      * @throws NullPointerException if the specified predicate is null
+     * @throws IllegalArgumentException if the predicate is a {@link com.hazelcast.query.PagingPredicate} or is a
```

Comment:
minor: I am not sure about our policy regarding the formatting of javadocs, but I see that others are formatted in a different way, the shorter exception names are padded with spaces so that the explanations start on the same column, and for explanations that are longer than a single line, subsequent lines start at the same column, not the first column, etc. Same for others